self_hosting_machinery/dashboard_service/utils.py

```diff
@@ -0,0 +1,90 @@
+import copy
+import pandas as pd
+
+from datetime import datetime
+from typing import Dict, Any
+from dataclasses import dataclass
+
+from self_hosting_machinery.webgui.selfhost_database import StatisticsService
+
+
+@dataclass
+class StatsDataTables:
+    network_df: pd.DataFrame
+    robot_human_df: pd.DataFrame
+    comp_counters_df: pd.DataFrame
+    extra: Dict
+
+
+class NoDataInDatabase(Exception):
+    pass
+
+
+def retrieve_all_data_tables(stats_service: StatisticsService) -> StatsDataTables:
+    extra = {}
+    network_df = pd.DataFrame(stats_service.network_select_all())
```

Comment:
We don't need all. We need for period, for last year from now

Reply:
@valaises is the author of this code, he can fix this